leadfoot/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+    "files": [
+        "index.d.ts",
+        "leadfoot-tests.ts",
+		"../node/index.d.ts",
```

Comment:
do not include files outside the module. use `/// <reference types="node" />` instead